components/BottomSheet/src/MDCBottomSheetPresentationController.h

```diff
@@ -64,6 +64,11 @@
  */
 @property(nonatomic, assign) BOOL dismissOnBackgroundTap;
 
+/**
+ This is used to set a custom height on the sheet view.
```

Comment:
Please expand on this a bit to document how it's used in conjunction with `preferredContentSize`. Specifically, it seems to override that value and is not (for instance) used as a "backup" value if the VC gives an invalid value `preferredContentSize`.

Please also document that it will be ignored if it's non-positive.

Reply:
Done